modules/00_3Scale_InstallUsingNoob/00_1_Lab_3scaleInstall_Lab.adoc

```diff
@@ -0,0 +1,128 @@
+:noaudio:
+:scrollbar:
+:toc2:
+:linkattrs:
+:data-uri:
+
+== Goals of the lab
+
+* Create nooba storage class using OpenShift container storage operator
```

Comment:
NooBaa

Reply:
@vbkgithub Can you do a find/relace on Nooba and change it to Noobaa. It looks wrong in a lot of places.

Reply:
Done. Please check the latest version. Do I need to submit pull request again to see the changes